### PR TITLE
Move service-python into domain-rrd

### PR DESCRIPTION
**Pull request by msnwatson**
## What was the problem?

`service-python` was its own Gradle project but it contains functionality relating to `domain-rrd`, so it should be contained within that directory

Associated tickets or Slack threads:

- https://github.com/department-of-veterans-affairs/abd-vro/issues/1509

## How does this fix it?[^1]
Moves all code associated with `service-python` into `domain-rrd` and removes references in Gradle project. 

## How to test this PR
- As this is not a logic change, and RRD code is no going to be used anymore, successful build and unit testing is sufficient for testing.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: service-python/featuretoggle/build.gradle
Comment (msnwatson) [inline service-python/featuretoggle/build.gradle:1]: this is not marked as moved for some reason but the same file exists in new location

File: service-python/assessclaimdc7101/build.gradle
Comment (msnwatson) [inline service-python/assessclaimdc7101/build.gradle:1]: this is not marked as moved for some reason but the same file exists in new location

File: service-python/assessclaimdc6602v2/src/lib/queues.py
Comment (msnwatson) [inline service-python/assessclaimdc6602v2/src/lib/queues.py:1]: this is not marked as moved for some reason but the same file exists in new location

**Comment by msnwatson** [comment]
Converting this back to draft for now, as a preceding step to this is removing the RRD containers out of the application docker-compose file

**Comment by yoomlam** [comment]
@msnwatson I think this PR just moves the folder, with no change in VRO build and run behavior. That will allow you to merge this PR. Then another PR will be created for disabling RRD functionality. Separate PRs will allow us to undo one without undoing the other.

**Review by yoomlam — APPROVED** [review]

**Review by yoomlam — APPROVED** [review]

**Comment by yoomlam** [comment]
You need to update [these lines](https://github.com/department-of-veterans-affairs/abd-vro/blob/239dc2724e1ccfc6e4f08bc324a35c9c057539e5/build.gradle#L162-L165)

**Review by tcraghu — APPROVED** [review]

**Pull request merged**